examples/mobile/UIComponents/components/notification-badge/number-badge-main-tab.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<meta content="width=device-width, user-scalable=no" name="viewport" />
+	<link href="../../lib/tau/mobile/theme/default/tau.css" rel="stylesheet" />
+	<link href="../../css/style.css" rel="stylesheet" />
+	<script data-build-remove="false" src="../../lib/tau/mobile/js/tau.js">
+	</script>
+</head>
+
+<body>
+	<div class="ui-page">
+		<div class="ui-content">
+			<!-- this page will be replaced by page from link target of active tab -->
+		</div>
+		<div class="ui-main-tab ui-active">
+			<ul>
+				<li class="ui-li-anchor">
+					<a class="ui-tab-active" href="one.html">
+						One
+					</a>
+				</li>
+				<li class="ui-li-anchor">
+					<a href="two.html">
+						Two
+					</a>
+				</li>
+				<li class="ui-li-anchor">
+					<a href="three.html">
+						Three
+					</a>
+				</li>
+				<li class="ui-li-anchor">
+					<a href="four.html">
+						Four
+						<span class="ui-badge"></span>
```

Comment:
Don't you need a table value?

Reply:
You mean `title` value? We don't need title value because this is only `dot`. I assumed that if a badge has no title it is a period. Developer any time can change element property to change badge look.